### PR TITLE
improve data version readability and update package.json

### PR DESCRIPTION
**Pull request by eleanor-byhook**
Looks more like a time stamp
UTC because timezones.

### CODE REVIEW

File: gulp/tasks/version.js [hunk 1]
```diff
@@ -72,7 +72,7 @@ module.exports = function( gulp, options ) {
     var version;
 
     if ( !args.version ) {
-      version = date.getTime();
+      version = date.getUTCMonth() + '/'+date.getUTCDate()+'/'+date.getUTCFullYear()+' - '+date.getUTCHours()+':'+date.getUTCMinutes()+':'+date.getUTCSeconds();
```
Comment (draeton) [inline gulp/tasks/version.js]: I would use `date.toISOString()`; it zero pads the single digit values.

**Comment by eleanor-byhook** [comment]
good call.  updated.

**Comment by draeton** [comment]
👍

**Comment by eleanor-byhook** [comment]
haven't had a chance to test the new package.json on windows, anybody with a windows machine want to try it out?

**Comment by draeton** [comment]
I get this error when running the `gulp` default task on Windows 10:

```
module.js:339
    throw err;
    ^

Error: Cannot find module 'jshint/src/cli'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (S:\home\projects\gulp-hook-standards-scaffold\node_modules\gulp-jshint\src\extract.js:1:79)
    at Module._compile (module.js:425:26)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
```

**Comment by eleanor-byhook** [comment]
how about now?

**Comment by eleanor-byhook** [comment]
just tested in a windows VM and with the latest commit it works, could someone else verify?

**Comment by herb-t** [comment]
this fires up no errors on my PC Win10

**Comment by eleanor-byhook** [comment]
Cool.  If anyone else wants to verify today, let me know, otherwise I'll merge it in EOD so I can submit another PR or two.

**Comment by eleanor-byhook** [comment]
@draeton, good to merge?

**Pull request merged**